docs/software/sciapps/cp2k.md

```diff
@@ -1,4 +1,400 @@
 [](){#ref-uenv-cp2k}
 # CP2K
-!!! todo
-    complete docs
+
+[CP2K] is a quantum chemistry and solid state physics software package that can perform atomistic simulations of solid
+state, liquid, molecular, periodic, material, crystal, and biological systems.
+
+CP2K provides a general framework for different modeling methods such as DFT using the mixed Gaussian and plane waves
+approaches GPW and GAPW. Supported theory levels include DFTB, LDA, GGA, MP2, RPA, semi-empirical methods (AM1, PM3,
+PM6, RM1, MNDO, …), and classical force fields (AMBER, CHARMM, …). CP2K can do simulations of molecular dynamics,
+metadynamics, Monte Carlo, Ehrenfest dynamics, vibrational analysis, core level spectroscopy, energy minimization, and
+transition state optimization using NEB or dimer method. See [CP2K Features] for a detailed overview.
+
+!!! note "User Environments"
+
+    [CP2K] is provided on [ALPS](#platforms-on-alps) via [User Environments](#ref-tool-uenv)
```

Comment:
these links are broken.

`[uenv][ref-tool-uenv]` (square braces, no `#`).

Also, I use `uenv` all lower case, not UENV or User Environments.